app/views/users/registrations/new.html.erb

```diff
@@ -1,19 +1,25 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l">Sign up</h2>
+    <h2 class="govuk-heading-l">Register your school</h2>
 
-    <%= form_for(resource, as: resource_name, url: users_register_path) do |f| %>
-      <%= f.govuk_error_summary %>
+    <%= form_for(:induction_coordinator_profile, url: users_check_details_path) do |f| %>
+      <%= f.govuk_text_field(
+            :school_urn,
+            value: @school&.urn,
+            hint: { text: "6 digit number", class: "govuk-hint" },
+            label: { text: "Your school's Unique Reference Number (URN)", class: "govuk-label govuk-label--s" })
+      %>
+      <p class="govuk-body govuk-!-font-weight-bold">
+        <%= govuk_link_to "Find my URN", "#", class: "govuk-link" %>
```

Comment:
Would be good to have a todo here so it doesn't get lost

Reply:
There was some talk of removing that altogether in the demo this morning. But yeah, I can add a todo.